### ISSUE TITLE
bug report for bestcheckpoint.pth.tar

### ISSUE THREAD

**Issue by kaenkogashi** [ASMIftekhar/VSGNet#8]
Dear Sir,

Thanks for your excellent work!
I followed the instructions and trained the COCO model from scratch.
After I got bestcheckpoint.pth.tar, I  tried to store the results in v-coco format, but I got "Failed to load checkPoint" error message.

>~/VSGNet/scripts$ CUDA_VISIBLE_DEVICES=0,1,2,3 python2 main.py -fw COCO_result -ba 8 -r t -i t
Doing a test run to detect bad detections

In training set object detector failed to detect any person in the following images:
[6357, 17487, 130851, 153892, 176148, 326601, 364400, 365013, 369213, 429514, 483039, 560726]
In validation set object detector failed to detect any person in the following images:
[130099, 196981, 352877, 387895, 426849, 526087, 567439, 568117]
In testing set object detector failed to detect any person in the following images:
[16875, 30828, 56701, 66706, 75768, 165157, 228418, 230501, 255483, 267725, 293855, 344045, 413805, 419143, 451038, 479280, 561411]
../COCO_result/bestcheckpoint.pth.tar
0
Failed to load checkPoint
Only doing testing for storing result from a model
Epoch 0/0

I noticed bestcheckpoint.pth.tar's data size(303633714  ) is a little different from other tar such as 20checkpoint.pth.tar(302550252  ). Is this is a bug or not? Thank you very much!
I only got "Failed to load checkPoint error " by  bestcheckpoint.pth.tar. Other tars are worked.

>-rw-rw-r-- 1  776297312  7月 30 21:44 test20.pickle
-rw-rw-r-- 1  302550252  7月 30 21:44 20checkpoint.pth.tar
-rw-rw-r-- 1  774514896  7月 31 01:40 test40.pickle
-rw-rw-r-- 1  302550252  7月 31 01:40 40checkpoint.pth.tar
-rw-rw-r-- 1  771836105  7月 31 05:40 test60.pickle
-rw-rw-r-- 1  302550252  7月 31 05:40 60checkpoint.pth.tar
-rw-rw-r-- 1  303633714  7月 31 07:50 bestcheckpoint.pth.tar
-rw-rw-r-- 1  768390928  7月 31 09:44 test80.pickle
-rw-rw-r-- 1  302550252  7月 31 09:44 80checkpoint.pth.tar
-rw-rw-r-- 1          6  7月 31 09:44 result.pickle

**Comment by kaenkogashi** [comment]
Already solved by myself. thank you very much!
